docs/docs/user/getting-started/components.md

```diff
@@ -19,13 +17,23 @@ Can be used to deploy any app in Kubernetes using helm. An example is a Redis da
     config:
     ```
 
-### StreamsApp
+### Commands
```

Comment:
I don't think we necessary need `Commands` here. It would be enough if we have `deploy`, `destroy`, `reset`, and `clean` section under each component

Reply:
changed

Reply:
tbh I dislike how it looks with the commands just floating out there, I don't think they deserve as big of a font as 'Configuration' or 'Usage'. 'Commands' may not be a correct title, but maybe they still could be brought together under a banner?

Reply:
how about "operations" or "methods"?

Reply:
"methods" sounds good to me

Reply:
What is wrong with commands? I actually think commands is pretty good and better than the alternatives

Reply:
I like "Commands" and "Methods" the same. Perhaps "commands" is what the user would call them while "methods" is more on the developer side.

Reply:
Method is a developer term while those are commands that a user can execute

Reply:
I almost want to argue that "operations" makes the most sense because of the name kpops (Kafka pipeline operations) :D but I really wouldn't mind going with any of these options

Reply:
That's a really good one, "Operations" would be very cool!